open_humans/management/commands/vacuum_log_bucket.py

```diff
@@ -0,0 +1,99 @@
+from datetime import datetime
+from urllib.parse import urlparse, parse_qs
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import boto3
+
+from data_import.models import DataFile, AWSDataFileAccessLog, NewDataFileAccessLog
+
+
+class Command(BaseCommand):
+    """
+    A management command vaccuming up file access logs and putting them in the database.
+    """
+
+    help = "Populates database with file access logs"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Retreiving file access logs")
+        s3 = boto3.resource("s3")
+        bucket = s3.Bucket(settings.LOG_BUCKET)
+        for item in bucket.objects.all():
+            log_objects = item.get()["Body"].read().decode("utf-8").split("\n")
+            for log_entry in log_objects:
+                if not log_entry:
+                    continue
+                log = []
+                start_chars = ['"', "["]
+                end_chars = ['"', "]"]
+                tmp_index = 0
+                for string in log_entry.split(" "):
+                    if string[0] in start_chars:
+                        if string[-1] not in end_chars:
+                            tmp_index = len(log)
+                    if (not tmp_index) or (tmp_index == len(log)):
+                        log.append(string)
+                        continue
+                    log[tmp_index] = "{0} {1}".format(log[tmp_index], string)
+                    if string[-1] in end_chars:
+                        tmp_index = 0
+
+                aws_log_entry = AWSDataFileAccessLog()
+                aws_log_entry.bucket_owner = log[0]
```

Comment:
This set of code could be redone?
1. an array ['bucket_owner', 'bucket', ...]
2. for loop over range(len(array))
3. setattr(aws_log_entry, array[i], log[i])